recipes/gnupg/gnupg_1.4.19.oe

```diff
@@ -6,8 +6,8 @@ EXTRA_OECONF += "--with-readline"
 
 EXTRA_OECONF += "ac_cv_sys_symbol_underscore=no --without-libusb"
 
-SRC_URI += "file://long-long-thumb.patch"
-SRC_URI += "file://mips_gcc4.4.patch"
+#SRC_URI += "file://long-long-thumb.patch"
+#SRC_URI += "file://mips_gcc4.4.patch"
```

Comment:
if these are not needed anymore, just remove them :) otherwise forward port is needed. The gcc-4.4 one at least seems not needed anymore since it builds fine